frontend/src/metabase/nav/containers/MainNavbar/MainNavbar.styled.tsx

```diff
@@ -63,11 +63,11 @@ export const NavRoot = styled.nav<{ isOpen: boolean }>`
   }
 
   ${breakpointMinSmall} {
-    width: ${NAV_SIDEBAR_WIDTH};
+    width: ${props => (props.isOpen ? NAV_SIDEBAR_WIDTH : 0)};
   }
 
   ${breakpointMaxSmall} {
-    width: 90vw;
+    width: ${props => (props.isOpen ? "90vw" : 0)};
```

Comment:
It was weird that the parent (`aside`) collapses while the child doesn't.
Context: https://metaboat.slack.com/archives/C505ZNNH4/p1727942364832079